tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -442,6 +442,35 @@ public async Task SetsExpectedProperties(SearchFilters searchFilters, string exp
 }", json);
             }
 
+            [Theory]
+            [MemberData(nameof(CatalogPackageTypesData))]
+            public void SetsExpectedPackageTypes(List<NuGet.Protocol.Catalog.PackageType> packageTypes, string[] expectedFilterable, string[] expectedDisplay)
+            {
+                var leaf = Data.Leaf;
+                leaf.PackageTypes = packageTypes;
+
+                var document = _target.UpdateLatestFromCatalog(
+                    SearchFilters.Default,
+                    Data.Versions,
+                    isLatestStable: false,
+                    isLatest: true,
+                    normalizedVersion: Data.NormalizedVersion,
+                    fullVersion: Data.FullVersion,
+                    leaf: leaf,
+                    owners: Data.Owners);
+
+                SetDocumentLastUpdated(document);
+                Assert.Equal(document.FilterablePackageTypes.Length, document.PackageTypes.Length);
+                Assert.Equal(expectedFilterable.Length, document.FilterablePackageTypes.Length);
```

Comment:
nit: you can do `Assert.Equal` with entire collections, avoiding the length assertion and loop